docker-src/db/Dockerfile

```diff
@@ -11,7 +11,7 @@ ENV LC_MESSAGES=en_US.UTF-8
 COPY sources/create_db.sh /docker-entrypoint-initdb.d/
 COPY sources/metasfresh.pgdump /tmp/
 
-#ENTRYPOINT ["/docker-entrypoint.sh"]
+ENTRYPOINT ["/usr/local/bin/docker-entrypoint.sh"]
```

Comment:
does this work?

Reply:
yes, otherwise docker-compose up -d would fail like
```
$ docker-compose up -d
Recreating docker-src_db_1 ...
Recreating docker-src_db_1 ... error

ERROR: for docker-src_db_1  Cannot start service db: failed to create shim: OCI runtime create failed: container_linux.go:380: starting container process caused: exec: "/docker-entrypoint.sh": stat /docker-entrypoint.sh: no such file or directory: unknown

ERROR: for db  Cannot start service db: failed to create shim: OCI runtime create failed: container_linux.go:380: starting container process caused: exec: "/docker-entrypoint.sh": stat /docker-entrypoint.sh: no such file or directory: unknown
Encountered errors while bringing up the project.

```